python/setup.py

```diff
@@ -19,10 +19,10 @@
         "pytest~=6.0",
         "pytest-xdist~=1.34",
         "scipy~=1.4",
-        "lib_logging",
+        #"lib_logging",
```

Comment:
What is the reason for removing this dependency on lib_logging?  Why was a dependency on lib_logging included in the first place since it contains zero python?

Reply:
This change snuck in accidentally - I removed it because it's not necessary, as you said it doesn't contain any python code. Happy to bring this change out into a separate PR for clarity.